Date/Time: 28/08/2019 23:10:27 [UTC+0] [GMT+0]
Operating System: Microsoft Windows NT 10.0.17134.0
Function: TestConnection
Message: Impossível conectar-se ao servidor remoto
Details: System.Net.WebException: Impossível conectar-se ao servidor remoto ---> System.Net.Sockets.SocketException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu 35.247.196.229:8000
   em System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   em System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fim do rastreamento de pilha de exceções internas ---
   em System.Net.HttpWebRequest.GetResponse()
   em Launcher_Updater.Launcher.TestConnection()
-
Date/Time: 28/08/2019 23:12:49 [UTC+0] [GMT+0]
Operating System: Microsoft Windows NT 10.0.17134.0
Function: TestConnection
Message: Impossível conectar-se ao servidor remoto
Details: System.Net.WebException: Impossível conectar-se ao servidor remoto ---> System.Net.Sockets.SocketException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu 35.247.196.229:8000
   em System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   em System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fim do rastreamento de pilha de exceções internas ---
   em System.Net.HttpWebRequest.GetResponse()
   em Launcher_Updater.Launcher.TestConnection()
-
